Infrastructure/GhostNetwork.Publications.MongoDb/MongoPublicationStorage.cs

```diff
@@ -25,6 +26,11 @@ public async Task<Publication> FindOneByIdAsync(string id)
 
             var filter = Builders<PublicationEntity>.Filter.Eq(p => p.Id, oId);
             var entity = await context.Publications.Find(filter).FirstOrDefaultAsync();
+            if (entity != null)
+            {
+                var comments = await context.Comments.Find(x => x.PublicationId == entity.Id.ToString()).ToListAsync();
```

Comment:
1) Avoid lambda queries. Use native query syntax
2) Each linq method should starts on new line. It's more readable
3) Publication can contains hundreds on comments. Fetching them all can have negative performance impact